File: c/bee1132.c
#include <stdio.h>
 
int main() {
    int x, y, soma=0;
    scanf("%d\n%d", &x, &y);

    if(x <= y) {
        for (int i=x; i <= y; i++) {
            if (i%13!=0) {
                soma += i;
            }
        }
    } else {
        for (int i=y; i <= x; i++) {
            if (i%13!=0) {
                soma += i;
            }
        }
    }
    
    printf("%d\n", soma);
    return 0;
}

File: python/bee1132.py
x = int(input())
y = int(input())
soma = 0

if x <= y:
    for n in range(x, y+1):
        if n % 13 != 0:
            soma += n
else:
    for n in range(y, x+1):
        if n % 13 != 0:
            soma += n
print(soma)
